breathe-and-relax-app/script.js

```diff
@@ -1,4 +1,4 @@
-const container = document.getElementsByClassName('container');
+const container = document.getElementsByClassName('container')[0];
```

Comment:
`document.getElementsByClassName` got an array of elements, not just one element. so what you needed to do was use the one element in this array.